framework/components/ecl-site-headers/ecl-site-headers.config.js

```diff
@@ -1,19 +1,16 @@
 module.exports = {
   title: 'Site headers',
   label: 'Site headers',
-  tags: ['organism'],
   status: 'ready',
-  variants: [
-    {
-      name: 'default',
-    },
-    {
-      name: 'homepage',
-      context: {
-        global: {
-          language: 'en',
-        },
-      },
-    },
-  ],
+  tags: ['organism'],
+  context: {
+    political: { href: '#', label: 'Commission and its priorities' },
+    info: { href: '#', label: 'Policies, information and services' },
+    title: 'User menu',
+    links: [
```

Comment:
`title`+ `links` could be grouped under `user_menu` to better distinguish them from the `political` and `info` links